File: CosmeticaShop.IServices/Models/Blog/BlogModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Web;

namespace CosmeticaShop.IServices.Models.Blog
{
    public class BlogModel
    {
        public int Id { get; set; }

        public DateTime DateCreate { get; set; }

        public string Title { get; set; }

        public string Content { get; set; }

        public string PreviewContent { get; set; }

        public string PhotoUrl { get; set; }

        public string KeyUrl { get; set; }

        public string SeoKeywords { get; set; }

        public string SeoDescription { get; set; }

        public bool IsActive { get; set; }

        public HttpPostedFileBase PhotoFile { get; set; }
    }
}

File: CosmeticaShop.Web/Areas/Admin/assets/js/Form/Blog/BlogModel.js
var Blog = Blog || {};

(function () {

    if (Blog.BlogModel) {
        console.error("Blog.BlogModel уже был создан");
        return;
    }

    /**
     * Класс для описания модели бренда
     * @param {     
     * } theParams 
     * @returns {} 
     */
    Blog.BlogModel = function (theParams) {
        theParams = theParams || {};

        this.Id = ko.observable(theParams.Id || "");
        this.Title = ko.observable(theParams.Title || "");
        this.PhotoUrl = ko.observable(theParams.PhotoUrl || "");
        this.KeyUrl = ko.observable(theParams.KeyUrl || "");
        this.IsActive = ko.observable(theParams.IsActive || false);


        return this;
    };

    /**
     * Определяем конструктор
     */
    Blog.BlogModel.prototype.constructor = Blog.BlogModel;

    Blog.BlogModel.prototype.log = function (text) {
        console.log("Ошибка в классе Blog.BlogModel: " + text);
    };

    Blog.BlogModel.prototype.GetData = function () {
        return {

        }
    }
})();